samples/plugins/mysql_odbc/connectionResolver.tdr

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<tdr class='mysql_odbc_sample'>
+  <connection-resolver>
+    <connection-builder>
+      <script file='connectionBuilder.js'/>
+    </connection-builder>
+    <connection-normalizer>
+      <required-attributes>
+        <attribute-list>
+          <attr>server</attr>
+          <attr>dbname</attr>
+          <attr>username</attr>
+          <attr>password</attr>
+          <attr>port</attr>
+          <attr>odbc-connect-string-extras</attr>
+          <attr>source-charset</attr>
+          <attr>one-time-sql</attr>
```

Comment:
This one is also preset.

Reply:
Removed `<attr>one-time-sql</attr>` in b07a30b